src/review/routes/profile/Profile.js

```diff
@@ -1,46 +1,97 @@
-import React, { PropTypes } from 'react'
+import React, { Component, PropTypes } from 'react'
 import { Link as LinkRouter } from 'react-router'
+import { connect } from 'react-redux'
 import Text from 'material-ui/Text'
-import ViewContainer from 'modules/components/ViewContainer'
+import Paper from 'material-ui/Paper'
+import {
+  List,
+  ListItem,
+  ListItemText,
+} from 'material-ui/List'
 import Link from 'modules/components/Link'
+import WatchTask from 'modules/components/WatchTask'
+import WatchTaskContainer from 'modules/components/WatchTaskContainer'
+import ViewContainer from 'modules/components/ViewContainer'
 import ScrollView from 'modules/components/ScrollView'
 import LayoutBody from 'modules/components/LayoutBody'
 import ReviewAppBar from 'review/modules/AppBar/AppBar'
+import actionTypes from 'review/modules/redux/actionTypes'
 
-function Profile(props) {
-  const {
-    profileName,
-  } = props.params
+class Profile extends Component {
+  static propTypes = {
+    dispatch: PropTypes.func.isRequired,
+    fetch: PropTypes.object.isRequired,
+    params: PropTypes.shape({
+      profileName: PropTypes.string.isRequired,
+    }).isRequired,
+  }
 
-  return (
-    <ViewContainer>
-      <ReviewAppBar />
-      <ScrollView>
-        <LayoutBody margin>
-          <Text type="display1" component="h2" gutterBottom>
-            <Link component={LinkRouter} to={`/${profileName}`}>
-              {profileName}
-            </Link>
-          </Text>
-          <ul>
-            <li>
-              <Text>
-                <Link component={LinkRouter} to={`/${profileName}/material-ui`}>
-                  {'material-ui'}
-                </Link>
-              </Text>
-            </li>
-          </ul>
-        </LayoutBody>
-      </ScrollView>
-    </ViewContainer>
-  )
-}
+  componentDidMount() {
```

Comment:
Not a good idea to mix view concern with data fetching concern if we want to switch from redux to another solution.